src/Components/ArticlePage.jsx

```diff
@@ -1,25 +1,47 @@
 import Card from 'react-bootstrap/Card'
 import Button from 'react-bootstrap/Button'
-import {useParams} from 'react-router'
-import {Link} from 'react-router'
+import {useParams, Link } from 'react-router'
+import {useEffect, useState}from 'react'
+import axios from 'axios'
+import CommentCard from './CommentCard'
 
-
-function ArticlePage ({articles}){
+function ArticlePage (){
     const {article_id} = useParams();
-    const article = articles.find((article) => article.article_id.toString() === article_id);
+    const [article, setArticle] = useState([])
+    const [comments, setComments] = useState([])
```

Comment:
Ideally there should also be a loading state to tell the user something is loading while the request is being sent off to your api